client/src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import GlobalStyles from './box/global'
+import Header from './components/header'
+
+const App:React.FC = () => {

```

Comment:
Right not it's not recommended (discoured) way to type React components in this way
https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/
it's better in this way:
```
const App = (): React.ReactNode =>{...}
```

Reply:
I understood about this way.

Reply:
I can't use React.ReactNode for App component because type of ReactNode is not valid for JSX.